src/app/app.component.ts

```diff
@@ -12,9 +12,7 @@ export class AppComponent implements OnDestroy {
 
   mobileQuery: MediaQueryList;
 
-  menu = ['find-journey', 'create journey', 'list journey'];
-
-  fillerNav = this.menu;
+  fillerNav = ['find-journey', 'create journey', 'list journey'];
```

Comment:
I don't think that is a good idea to use the same variable for the link and for the name displayed for the link.

Reply:
right we need a refacto on that